<launch>
  <include file="$(find simcontrol)/launch/sim.launch">
    <arg name="use_ir_markers" value="true"/>
    <arg name="render_stereo" value="false"/>
  </include>
</launch>
